_data/changelogs/template-documentation.yml

```
@@ -0,0 +1,16 @@
+title: Documentation page
+type: template
+items:
+  - date: NNNN-NN-NN
+    summary: Visual order of the Documentation page matches the HTML order.
```

Recommend tweaking this a bit to follow the standard format and improve clarity.
```suggestion
    summary: Updated the order of the side navigation markup to match the visual order. 
```